packages/@expo/metro-config/src/serializer/__tests__/withExpoSerializers.test.ts

```diff
@@ -349,10 +528,11 @@ describe('serializes', () => {
         sourceMaps: false,
       },
     });
-    expect(typeof str).toBe('string');
+    expect(typeof bundle).toBe('object');
     // Ensure the module is run.
-    expect(str).toMatch(/TEST_RUN_MODULE\("\/app\/index\.js"\);/);
-    expect(str).toMatch(/expo-mock\/async-require/);
+    expect(bundle.code).toMatch(/TEST_RUN_MODULE\("\/app\/index\.js"\);/);
+    expect(bundle.code).toMatch(/expo-mock\/async-require/);
+    expect(bundle.map).toMatch(/debugId/);
```

Comment:
This is unexpected to me, why is the source map with debug ID present when the `sourceMaps: false` option is set?

Reply:
Is this the `export:embed` scenario?

Reply:
This is an issue with how `export:embed` works, the more recent versions don't perform additional work.